utils/build/docker/nodejs/parametric/package.json

```diff
@@ -10,6 +10,8 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@opentelemetry/api": "^1.9.0",
+    "@opentelemetry/core": "^1.25.1",
```

Comment:
Because the app uses it, but it previously used the ones installed transitively by `dd-trace`, which is an anti-pattern. An app should always depend on all the modules it uses directly. This is especially true now that `dd-trace` can be linked, meaning its dependencies are no longer flattened at the root.